packages/core/src/plugins/DropPlugin.ts

```diff
@@ -0,0 +1,13 @@
+import { Plugin, PluginKey } from '@tiptap/pm/state'
+
+export const DropPlugin = (onPaste: (e: DragEvent) => void) => {
+  return new Plugin({
+    key: new PluginKey('tiptap__drop'),
```

Comment:
Maybe keep this in lowerCamelCase (`tiptapDrop`), like we do in other plugins?

Reply:
Or maybe call it `dropEvent`, analog to `focusEvents`?

Reply:
how about `tiptap__dropEvent` - I'm using the `tiptap__` prefix to make sure to never accidentially conflict with other pluginKeys

Reply:
PluginKeys are symbols by default no? So this should only ever exist once.